app/assets/javascripts/auto_title.js

```diff
@@ -0,0 +1,24 @@
+function onTitleRetrievalSuccess(data, status, xhr) {
+  // $(#bookmark_title).val(data);
```

Comment:
When I try using this commented-out line I get "Uncaught SyntaxError: Unexpected token ILLEGAL".  Research suggests it may have to do with an invisible character of some sort.